modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -131,5 +329,13 @@ function _ding_webtrekk_set_data_layer_values(array $value_map) {
   $script = implode(";\n", $script_lines);
   drupal_add_js($script, [
     'type' => 'inline',
+    'group' => JS_LIBRARY,
+    'weight' => -22,
```

Comment:
A comment about the weight here would be good.

Reply:
Agree, will add comment.